src/Model/ValueObject/Identity/UUID.php

```diff
@@ -6,25 +6,33 @@
 
 use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\IsString;
 use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\MatchesRegexPattern;
+use Ramsey\Uuid\Uuid as RamseyUuid;
 
 class UUID
 {
     use IsString;
     use MatchesRegexPattern;
 
+    public const NIL = '00000000-0000-0000-0000-000000000000';
```

Comment:
There is also `LocationId::NIL_LOCATION` and some hardcoded strings, maybe we need to make that consistent?